### DISCUSSION TITLE
Enable server-side decoration for Linux

### DISCUSSION THREAD

**Discussion by ghost** [standardnotes/forum#2596] [Suggestions]
Is it possible to disable the client-side decorations for Standard Notes on Linux? Meaning I can use the system-wide title bar? (Similar to use system title bars and borders in Google Chrome) The app looks completely out of place on my desktop. This is the only thing that's keeping me from getting Extended, because I like all other features of Standard Notes.

**Comment by atmoio** [comment]
It's not possible yet, but this is an option we'll likely add in a future update.
